backend/app/services/group_service.py

```
@@ -254,6 +256,68 @@ def list_user_groups(
+    Args:
+        db: Database session
+        q: Search query string
+        skip: Number of records to skip
+        limit: Maximum number of records to return
+        user_id: Current user ID (optional — filters to user's groups if provided)
+        user_role: Current user role (optional)
+    Returns:
+        Tuple of (list of GroupResponse objects, total count)
+    """
+    query = db.query(Namespace).filter(
+        Namespace.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)